resolvers/index.js

```diff
@@ -137,4 +140,35 @@ module.exports = {
     siteRecord.fields.InventoryUpdates = inventoryUpdates;
     return siteRecord;
   },
+  [Tables.PurchaseRequests]: {
+    write: async (purchaseRequestRecord, authRecord) => {
+      // Only allow admin users to review purchase requests
+      if (
+        purchaseRequestRecord.fields.hasOwnProperty("Status") &&
+        purchaseRequestRecord.fields.Status !== "Pending"
+      ) {
+        if (
+          authRecord.fields.Admin &&
+          purchaseRequestRecord.fields.hasOwnProperty("Reviewer") &&
+          purchaseRequestRecord.fields.Reviewer[0] === authRecord.id
+        ) {
+          return purchaseRequestRecord;
+        } else {
+          console.log("[Purchase Requests] Review access denied");
```

Comment:
NIT: specify something related to admin access so future devs debugging can see potential reason